exporters/otlp/src/main/java/io/opentelemetry/exporter/otlp/OtlpGrpcMetricExporter.java

```diff
@@ -153,11 +153,12 @@ public static OtlpGrpcMetricExporter getDefault() {
    * cancelled. The channel is forcefully closed after a timeout.
    */
   @Override
-  public void shutdown() {
+  public CompletableResultCode shutdown() {
     try {
       managedChannel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
     } catch (InterruptedException e) {
       logger.log(Level.WARNING, "Failed to shutdown the gRPC channel", e);
```

Comment:
should we return a failure in this case?

Reply:
yes. missing that. will fix it, thanks. : )